sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/SendSmsToGroupExample.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.sms.samples.quickstart;
+
+import com.azure.communication.sms.SmsClient;
+import com.azure.communication.sms.SmsClientBuilder;
+import com.azure.communication.sms.models.SmsSendOptions;
+import com.azure.communication.sms.models.SmsSendResult;
+import com.azure.core.util.Context;
+import java.util.Arrays;
+
+public class SendSmsToGroupExample {
+    public static void main(String[] args) {
+        String connectionString = System.getenv("COMMUNICATION_CONNECTION_STRING");
+        String phoneNumber = System.getenv("COMMUNICATION_PHONE_NUMBER");
```

Comment:
Do the samples need to get the phoneNumber from environment variables? It is needed for our live tests but I think it just adds additional complexity in this case.

Reply:
they will need to once samples start getting run in the pipeline, but we're still finalizing our approach

Reply:
make the sample just 1:1 message to make it simpler